### PR TITLE
Add package gui-libs/aquamarine

### PR DESCRIPTION
**Pull request by JohnMH**
Aquamarine is a new dependency of Hyprland, and is needed for live builds.

Reqires #37676.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by JohnMH** [comment]
@MrRoy Added you to the list of maintainers as requested on IRC.

**Comment by JohnMH** [comment]
> Having multi commits(packages) in one single PR is just fine, and it's actually easy for reviewer to check all the dependencies, in this case gui-libs/aquamarine depend on dev-util/hyprwayland-scanner. And I'd suggest just sending one PR for all hyprland related packages (instead of 4 PRs).

Sure, I can either close these and submit them as one now or do that in the future. Whichever you prefer. This was separated at ztrawhcse's recommendation on IRC.

> Adding yourself to maintainer list means you're willing to work with the packages (hyprland) in the long term, not just doing the version bump, but also handling bugs, stable request, security issues etc.. if you're not ready, please don't do this. Even you're fine to help, please adding yourself after have several contributions, do show yourself having interests/commitment/endeavor. Also I'd like to see explicit approval from MrRoy for this

I fully understand. I already maintain parts of the Hyprland ecosystem in GURU. MrRoy mentioned looking over these tonight on IRC. If you'd like, I can remove the commits adding myself to the maintainers file.

**Comment by eli-schwartz** [comment]
> Sure, I can either close these and submit them as one now or do that in the future. Whichever you prefer. This was separated at ztrawhcse's recommendation on IRC.

This is a misunderstanding of what I said -- I commented, in response to:

```
14:23 <MrRoy> JohnMH: any particular reason to bump it (bug, vuln, etc)? I usually wait until there's a hyprland release to bump the deps
```

```
14:27 <+ztrawhcse> i can think of worse tragedies than providing updated packages with new improvements, but done one package at a time instead of tons of changes all at once 
```

Specifically, it's okay to submit a PR to update a package immediately, rather than wait until its reverse dependencies start to require it. Updating a package when it's the *only* package with an available update, is still valuable.

If you have multiple related updates pending, it makes sense to batch them together for convenience.

And in this case, where aquamarine depends on another PR, it is *mandatory* to do one of:
- batch it together in the same PR as its dependency
- provide complicated, handwritten instructions regarding which PRs should be *ignored* because they are, currently, invalid PRs that break the repository CI and would result in emerge failing if they were actually merged

tl;dr

I apologize for leading you astray, I should have communicated more clearly.
